stats/utils.go

```diff
@@ -1,54 +0,0 @@
-package stats
```

Comment:
It might be better to add a comment that the `stats` package has been deprecated in favor of `crit` and will be removed in the next major release.